manifests/0000_50_cluster-platform-operator-manager_07-core-clusteroperator.yaml

```diff
@@ -0,0 +1,28 @@
+apiVersion: config.openshift.io/v1
+kind: ClusterOperator
+metadata:
+  annotations:
+    exclude.release.openshift.io/internal-openshift-hosted: "true"
+    include.release.openshift.io/self-managed-high-availability: "true"
+    include.release.openshift.io/single-node-developer: "true"
+    release.openshift.io/feature-set: TechPreviewNoUpgrade
+  name: platform-operators-core
+  namespace: openshift-platform-operators
```

Comment:
Should we be putting a namespace in the CO metadata? Aren't these cluster-scoped objects? Is this just an artifact of kustomize namespace injection? I wonder if there's a way to teach kustomize about the scoping of resources so that it doesn't inject namespaces into cluster-scoped objects.

Reply:
Yup, this is just an artifact of kustomize injecting these fields. This is also a problem with the aggregate CO YAML manifest. AFAIK, it's a no-op when deployed at runtime.

Reply:
It is, but it could be confusing to someone looking at this or the contents of the release image. Maybe something to try to follow-up on later.

Reply:
Works for me 👍

Reply:
If we wanted to, we could put our ClusterOperator files into a separate folder and render them with a separate kustomize call. That would make it so that we can avoid this issue with a little bit of a hack around it.

Reply:
I feel like there's gotta be a kustomize-centric way to handle this. Does `kustomize` add a `.metadata.namespace` to `Namespace` objects?

Reply:
Yeah, there is a part of our kustomize file that is telling it to add namespaces to all resources.

https://github.com/openshift/platform-operators/blob/4ecea427cf5302dfcdf4a5af8d28eadebacc2037/config/default/kustomization.yaml#L2

I'm also sure there is a kustomize centric way to tell it "add namespaces to these, but not these resources"